osh/hub/templates/waiving/list.html

```diff
@@ -8,7 +8,9 @@ <h2>{% trans title %}</h2>
 {{ search_form.release.label }} {{ search_form.release }}
 {{ search_form.search }}
 <input type="submit" value="{% trans "Search" %}" />
-{% if user.is_authenticated %}<br />{{ search_form.my }} Show only my scans{% endif %}
+{% if user.is_authenticated %}
```

Comment:
Should we rather move the `<br/>` tag before the condition?  The `Show only my tasks` checkbox at `/task/` is also shown on a separate line.

Reply:
But `/scan/packages/ shows everything on the same line... I guess it does not really matter as long as the single view itself stays consistent.